src/converters/replugged.ts

```diff
@@ -1,15 +1,32 @@
 import { ClassDeclaration, ClassMethod, ExportDefaultDeclaration, Identifier, Statement } from "@babel/types";
 import { IModImplementation } from "../api/ModImplementation.js";
-import { createFunctionWithWrapperNeeded } from "../api/RuntimeGenerators.js";
+import { createFunctionFromObjectProperty, createFunctionWithWrapperNeeded } from "../api/RuntimeGenerators.js";
 import { IBaseWebpackApi } from "../api/Webpack.js";
 import { parse } from "@babel/parser";
+import { IBasePatcherApi } from "../api/Patcher.js";
 
 class RPWebpackApi implements IBaseWebpackApi {
     get getModule() {
         return createFunctionWithWrapperNeeded("replugged.webpack", "getModule", "getModuleRawToExportedWrapper");
     }
 }
 
+class RPPatcherApi implements IBasePatcherApi {
+    get constructor_() {
+        return createFunctionFromObjectProperty("replugged", "Injector") as any;
+    }
+    constructor() {
+        return this.constructor_;
+    }
+    internalId: undefined;
```

Comment:
Add a comment to explain this for rp and betterdiscord

Reply:
The comment is in [`src/api/Patcher.ts`](https://github.com/LosersUnited/DiscordModCompiler/blob/a25154098a7cb7806f9d3abe6a4cc13db25adfa4/src/api/Patcher.ts#L2)